spring-cloud-dataflow-server/docker-compose-k8s.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+
+  skipper-server:
+    environment:
+      - SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_KUBERNETES_ACCOUNTS_K8S_FABRIC8_MASTER_URL=<MASTER URL>
+      - SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_KUBERNETES_ACCOUNTS_K8S_FABRIC8_USERNAME=<USER>
```

Comment:
This assumes Basic-auth is enabled at the cluster level, right?

Reply:
Also, by default, Basic auth is disabled in GKE at least.

Reply:
I have successfully tested it with K8s/GKE cluster. Did not experienced any issues.
Furthermore this is a basic setup/template to show how you can do it.